lib/stat_tracker.rb

```diff
@@ -134,9 +134,30 @@ def worst_offense #issue # 12
   end
 
   def highest_scoring_visitor #issue # 13
+      away_team_ids_array = (games[:away_team_id]).uniq.sort
 
+      team_ids_hash = {}
+      away_team_ids_array.each do |teamid|
+        team_ids_hash[teamid] = {sum_away_goals: 0, count_of_away_games_played: 0}
+      end
 
+      games.each do |row|
+        team_ids_hash[row[:away_team_id]][:sum_away_goals] += row[:away_goals]
+      end
+    
+      games.each do |row|
+        team_ids_hash[row[:away_team_id]][:count_of_away_games_played] += 1
+      end
```

Comment:
I think you could combine these two each methods into the same each method. Something like:
`games.each do |row|
        team_ids_hash[row[:away_team_id]][:sum_away_goals] += row[:away_goals]
         team_ids_hash[row[:away_team_id]][:count_of_away_games_played] += 1
      end`

Reply:
Awsome looks great that way it does both tings at one time rather then multiple iterations